From 85a69f948eb44575f728f0007760a4dabaadcb6f Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 11:38:15 +0800
Subject: [PATCH] MLK-10783-2 ARM: imx6ul-evk: Enable USB function

Since USBOTG1's power enable pin is conflict with Touch Screen,
and Touch Screen is the default feature, so USBOTG1 is pheripheral
only function, and USBOTG2 is host only function.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index 270c87f..b5c9f97 100644
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -127,6 +127,16 @@
 	status = "okay";
 };
 
+&usbotg1 {
+	dr_mode = "peripheral";
+	status = "okay";
+};
+
+&usbotg2 {
+	dr_mode = "host";
+	status = "okay";
+};
+
 &usdhc1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-- 
1.8.0

